### PR TITLE
w, wc: add Polish translation

### PR DESCRIPTION
**Pull request by chanibal**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/osx/w.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# w
+
+> Pokazuje kto jest zalogowany i co aktualnie robi.
+> Wyświetla login, TTY, zdalny host, czas zalogowania, czas bezczynności i aktualny proces.
+> Więcej informacji: <https://manned.org/w>.
```
Comment (navarroaxel) [inline pages.pl/osx/w.md:5]: The osx version for `w` doesn't have long arguments, only 1 letter.
```suggestion
> Więcej informacji: <https://ss64.com/osx/w.html>.
```
Please add this link to the main page too.

File: pages.pl/osx/wc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# wc
+
+> Zlicza linie, słowa, i bajty
+> Więcej informacji: <https://www.gnu.org/software/coreutils/wc>.
+
+- Policz linie w pliku
+
+`wc -l {{plik}}`
```
Comment (navarroaxel) [inline pages.pl/osx/wc.md:8]: Could you apply this change to `path/to/file` in the main page?
```suggestion
`wc -l {{ścieżka/do/pliku}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.pl/osx/wc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> Zlicza linie, słowa, i bajty
+> Więcej informacji: <https://www.gnu.org/software/coreutils/wc>.
+
+- Policz linie w pliku
+
+`wc -l {{ścieżka/do/pliku}}`
+
+- Policz słowa w pliku:
+
+`wc -w {{plik}}`
```
Comment (navarroaxel) [inline pages.pl/osx/wc.md:12]:
```suggestion
`wc -w {{ścieżka/do/pliku}}`
```

File: pages.pl/osx/wc.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Policz znaki (bajty) w pliku:
+
+`wc -c {{plik}}`
```
Comment (navarroaxel) [inline pages.pl/osx/wc.md:16]:
```suggestion
`wc -c {{ścieżka/do/pliku}}`
```

File: pages.pl/osx/wc.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Policz znaki w pliku (uwzględniając znaki zapisane więcej niż jednym bajtem):
+
+`wc -m {{plik}}`
```
Comment (navarroaxel) [inline pages.pl/osx/wc.md:20]:
```suggestion
`wc -m {{ścieżka/do/pliku}}`
```

File: pages.pl/osx/wc.md (hunk 2 above)
Comment (chanibal) [reply pages.pl/osx/wc.md]: Yes, I can, but the osx English TLDR page of `wc` also says `file`, and not `path/to/file`. 

If you always want the longer version suggesting that files can have a path, then this is not the only place where I translated `{{file}}` to `{{plik}}`.

Comment (navarroaxel) [reply pages.pl/osx/wc.md]: yeah, we are making a migration if we refresh an old page.

**Review by navarroaxel — APPROVED** [review]

**Comment by chanibal** [comment]
Updated the source TLDR pages with `path/to/file` and the ss64 manpage link.

File: pages/osx/w.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 > Show who is logged on and what they are doing.
 > Print user login, TTY, remote host, login time, idle time, current process.
+> More information: <https://ss64.com/osx/w.html>
```
Comment (navarroaxel) [inline pages/osx/w.md:5]:
```suggestion
> More information: <https://ss64.com/osx/w.html>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @chanibal

**Pull request merged**